packages/mobile/src/walletConnect/v1/saga.ts

```diff
@@ -69,7 +70,8 @@ export function* acceptSession(session: AcceptSession) {
   connector.approveSession(sessionData)
   connector.updateSession(sessionData)
   yield put(storeSession(connector.session))
-  yield call(handlePendingState)
+  yield put(showMessage(i18n.t('walletConnect:connectionSuccess', { dappName: peerMeta.name })))
```

Comment:
In the future we should introduce a more meaningful success action handled by the stores interested in it.